### ISSUE TITLE
Casing of enums

### ISSUE THREAD

**Issue by manu-st** [improbable-eng/ts-protoc-gen#21]
If I have the following definition:

```
message MyMsg
{
   enum MyEnum
   {
      None = 0;
      Manual = 1;
      Auto = 2;
   }
   MyEnum type = 1;
}
```

It generates the following TypeScript definition:

```
export namespace MyMsg{
  export type AsObject = {
    type: MyMsg.MyEnum,
  }

  export enum MyEnum {
    None = 0,
    Manual = 1,
    Auto = 2,
  }
}
```

However at runtime, the runtime object for MyEnum has values that are all uppercase, not what specified in the TypeScript definition. Shouldn't ts-protoc-gen generates the uppercase names to match what is generated?

**Comment by jonnyreeves** [comment]
Indeed, `protoc` will convert the case of enums into ALL_CAPS; the fix for ts-protoc-gen is straightforward, but I would also like to confirm that this behaviour in protoc is intentional so I've raised https://github.com/google/protobuf/issues/3551.
